tools/twix/src/panels/map/mod.rs

```diff
@@ -78,6 +78,7 @@ pub struct MapPanel {
     feet_detection: EnabledLayer<layers::FeetDetection, Ground>,
     ball_filter: EnabledLayer<layers::BallFilter, Ground>,
     obstacle_filter: EnabledLayer<layers::ObstacleFilter, Ground>,
+    ball_search_heatmap: EnabledLayer<layers::BallSearchHeatmap, Field>,
```

Comment:
is it wise to draw such a large feature last?

Reply:
You are right. I shifted it to be painted after lines.